application/Profile/DBProfile.py

```
@@ -85,8 +97,13 @@ class User(db.Model):
 
             @validates('username')
             def validate_username(self, key, username):
-                assert re.match("^[A-Za-z0-9_-]+$", username), \
-                    'User name should contain only letters, numbers, underscores and dashes'
+                username_error = 'User name should contain only letters, numbers, underscores and dashes'
+                try:
+                    assert re.match("^[A-Za-z0-9_-]+$", username)
```

Ruff

Use of `assert` detected

(S101)